ui/explore/src/components/PrometheusMetricsFinder/display/list/MetricList.tsx

```diff
@@ -42,7 +42,7 @@ export function MetricRow({ metricName, datasource, filters, isMetadataEnabled,
         <Typography sx={{ fontFamily: 'monospace' }}>{metricName}</Typography>
       </TableCell>
 
-      <TableCell style={{ minWidth: 'fit-content', textAlign: 'center' }}>
+      <TableCell style={{ width: 115, textAlign: 'center' }}>
```

Comment:
I guess that should depend on the size of the screen. At least on a phone, that should be different don't you think ?

Reply:
No, the content size is the same. The chip / button size will not change if it's a mobile or not 😄